src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const parentElement = document.querySelector('ul');
+const employees = Array.from(document.querySelectorAll('li'));
+
+function getSalary(employee) {
+  const salaryString = employee.getAttribute('data-salary');
+  const cleanSalary = salaryString.replace(/[^0-9.-]+/g, '');
```

Comment:
The regular expression used here removes all non-numeric characters, which is good for cleaning currency symbols. However, ensure that the salary data is always in a format that can be parsed by 'parseInt'. Consider using 'parseFloat' if the salary can be a decimal.